CHANGES.md

```diff
@@ -1,6 +1,9 @@
 0.6 (unreleased)
 ----------------
 
+- No changes yet.
```

Comment:
Can you remove the 'no changes yet'?

Reply:
so this was not put by me - but by @pllim

Reply:
Feel free to remove it now that there is a change. :)